imsim/stamp.py

```diff
@@ -573,10 +614,10 @@ def fix_seds(prof):
             gal = gal.evaluateAtWavelength(bandpass.effective_wavelength)
             gal = gal * self._trivial_sed
         else:
-            fix_seds(gal)
+            self._fix_seds(gal, bandpass)
         gal = gal.withFlux(self.realized_flux, bandpass)
 
-        wcs = base['wcs']
+        image.wcs = base['wcs']
```

Comment:
There should be a comment here describing why this is needed and why passing `wcs` as a keyword arg to `.drawImage` has problems.